pom.xml

```diff
@@ -1674,7 +1674,7 @@
                                 <configuration>
                                     <rules>
                                         <requireJavaVersion>
-                                            <version>[1.8,)</version>
+                                            <version>[1.9,)</version>
```

Comment:
This rule is incorrect. We don't support JDK 9, only 1.8.

Reply:
Resolved.